### DISCUSSION TITLE
Just because you can doesn’t mean you should.

### DISCUSSION THREAD

**Discussion by fungilation** [FerretDB/FerretDB#2629] [General]
Nosql on top of real sql db. [Really?](https://twitter.com/garyfung/status/1657495306576232448) Re: www.infoq.com/news/2023/05/ferretdb-mongodb-ga/

Sql is great. Be lazy, don’t reinvent joins in app logic. When you actually need to limit yourself for scaling/performance sake, use a real KV store like Redis

**Comment by ptrfarkas** [comment]
> Nosql on top of real sql db. [Really?](https://twitter.com/garyfung/status/1657495306576232448) Re: [www.infoq.com/news/2023/05/ferretdb-mongodb-ga/](http://www.infoq.com/news/2023/05/ferretdb-mongodb-ga/)
> 
> Sql is great. Be lazy, don’t reinvent joins in app logic. When you actually need to limit yourself for scaling/performance sake, use a real KV store like Redis

SQL is indeed great, and we are not working on making SQL any worse or less popular. We are working on creating options for those who prefer MongoDB.

**Comment by masterbater** [comment]
> Nosql on top of real sql db. [Really?](https://twitter.com/garyfung/status/1657495306576232448) Re: www.infoq.com/news/2023/05/ferretdb-mongodb-ga/
> 
> Sql is great. Be lazy, don’t reinvent joins in app logic. When you actually need to limit yourself for scaling/performance sake, use a real KV store like Redis

The best use case for this is you have a legacy mongodb project then moving forward your team needs to use postgresql.

I am evaluating ferretdb if it can pass all our test
